gui/options.cpp

```diff
@@ -2612,29 +2690,22 @@ void GlobalOptionsDialog::apply() {
 
 	bool isRebuildNeeded = false;
 
-	Common::U32String savePath(_savePath->getLabel());
-	if (!savePath.empty() && (savePath != _("Default")))
-		ConfMan.set("savepath", savePath.encode(), _domain);
-	else
-		ConfMan.removeKey("savepath", _domain);
-
-	Common::U32String themePath(_themePath->getLabel());
-	if (!themePath.empty() && (themePath != _c("None", "path")))
-		ConfMan.set("themepath", themePath.encode(), _domain);
-	else
-		ConfMan.removeKey("themepath", _domain);
-
-	Common::U32String iconPath(_iconPath->getLabel());
-	if (!iconPath.empty() && (iconPath != _c("None", "path")))
-		ConfMan.set("iconspath", iconPath.encode(), _domain);
-	else
-		ConfMan.removeKey("iconspath", _domain);
-
-	Common::U32String extraPath(_extraPath->getLabel());
-	if (!extraPath.empty() && (extraPath != _c("None", "path")))
-		ConfMan.set("extrapath", extraPath.encode(), _domain);
-	else
-		ConfMan.removeKey("extrapath", _domain);
+	const auto changePath = 
+	[&](GUI::StaticTextWidget *const widget, const Common::String& pathType, const Common::U32String &defaultLabel){
```

Comment:
Please fix formatting `const Common::String &pathType, const Common::U32String &defaultLabel) {`

wrong space around`&pathType` and missing space before `{`